Automattic/woocommerce-payments
# Merge progressive onboarding prototype under a feature flag

Fixes #5506 

#### Changes proposed in this Pull Request
Merge the existing progressive onboarding prototype branch behind `_wcpay_feature_progressive_onboarding` feature flag.

This includes all the changes from #4826, and the new feature flag condition.

#### Testing instructions
>**Note**: Get the latest WCPay dev tool code.
To avoid creating a new account you can replace [this return](https://github.com/Automattic/woocommerce-payments/blob/79f7e6308ad86ea571b5bfb5548f4e2bd79973f1/includes/class-wc-payments-account.php#L1479) with a `die`.

- Go to **WCPay Dev** and enable `Force the plugin to act as disconnected from WCPay`.
- Go to **Payments** and click on `Finish setup`
- Should redirect to Stripe or white page if you've replaced the `return`.
- Go to **WCPay Dev** and enable `Enable Progressive Onboarding`.
- Go to **Payments** and click on `Finish setup`
- Should redirect to the onboarding page, which contains a connect button and a form.
- Clicking on `Connect` on the top notice should allow you to onboard as usual.
- Filling the form and clicking on Submit should redirect to Stripe with all the details prefilled and progressive onboarding enabled.

-------------------


**Post merge**
